packages/ckeditor5-engine/src/model/model.ts

```diff
@@ -689,6 +689,7 @@ export default class Model extends ObservableMixin() {
 			doNotResetEntireContent?: boolean;
 			doNotAutoparagraph?: boolean;
 			direction?: 'forward' | 'backward';
+			[ i: string ]: unknown;
```

Comment:
Note: There is a need in CF/TC to add additional params eg. `forceDefaultExecution: true` in `deleteContent`.  Change of the contract is influenced by the change of `insertContent` where the function contract was extended.